fred2/management.cpp

```diff
@@ -1911,77 +1911,80 @@ int advanced_stricmp(char *one, char *two)
 // returns 0: go ahead change object
 //			  1: don't change it
 //			  2: abort (they used cancel to go to reference)
-int reference_handler(char *name, int type, int obj)
+int reference_handler(char *name, SEXP_REF_TYPE type, int obj)
 {
 	char msg[2048], text[128], type_name[128];
 	int r, n, node;
+	SEXP_SRC source;
```

Comment:
Why declare source up here?
Declarations at the function start are obsolete anyways, so I'd suggest moving it and the declaration for n down to when they're assigned.

Reply:
To be consistent with the other variables, but changed